tox.ini

```diff
@@ -0,0 +1,32 @@
+[tox]
```

Comment:
Maybe you could also add black-format check which will autoformat the code to be compliant with black. Though it shouldn't run by default (it shouldn't be part of the envlist)